cli/status/status.go

```diff
@@ -0,0 +1,62 @@
+package status
+
+import (
+	"fmt"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/fatih/color"
+	"github.com/tarantool/tt/cli/process_utils"
+	"github.com/tarantool/tt/cli/running"
+)
+
+const (
+	ColorSeqLen = 9
+	RowIndent   = 5
+	kInstance   = "INSTANCE"
```

Comment:
No need "k" prefix for consts.
I'd like to see from these three var names that they are related to the table header.
Alternatively, we can create 'columns' or 'header' array. So the purpose of these names and supposed columns order will be more clear. And use `Join` with tabs for this array instead of printf with constant names.

Reply:
Replaced these consts with a 'header' array and used `Join` to format.